### PR TITLE
Improve content for the "architecture" section

### PR DESCRIPTION
**Pull request by fakela**
Refined the content to improve grammar and clarity in the description of shard architecture.

### CODE REVIEW

File: docs/introduction/architecture.md [hunk 1]
```diff
@@ -3,51 +3,46 @@ title: Shard Architecture
...
-On top of that, Private shards use the NEAR account model inside the shard and will generate FunctionCall keys for specific contract interaction improving on security for the users and network.
+Calimero Shard is integrated with the NEAR wallet, which means that users authenticate using their NEAR Mainnet or Testnet FullAccess Key. Users sign an authorization message with the FullAccess Key to request permission from the Authorization Service for a specific shard. If they are granted permission, they receive a JWT token for communicating with the Private Shard. This authentication process ensures that only users with the corresponding key can issue transactions on the shard. Additionally, Private shards employ the NEAR account model within the shard and generate FunctionCall keys for specific contract interactions, enhancing security for users and the network.
```
Comment (emojapaula) [inline docs/introduction/architecture.md:46]: Additionally, private shards employ the NEAR account model within the shard and generate FunctionCall keys for specific contract interactions, enhancing security for users and the network.


Can we maybe link to https://docs.near.org/concepts/basics/accounts/access-keys#function-call-keys

or write a paragraph ourselves

**Review by emojapaula — CHANGES_REQUESTED** [review]

File: docs/introduction/architecture.md [hunk 2]
```diff
@@ -3,51 +3,46 @@ title: Shard Architecture
...
+In a proof of authority (PoA) shard, all validators have equal voting power and an equal number of blocks. PoA shards are commonly used in private or permissioned blockchain networks where validators are known and trusted entities, such as consortium members or enterprise partners. PoA shards offer faster transaction speeds and lower energy consumption compared to proof of work (PoW) or proof of stake (PoS) consensus mechanisms.
 
 ## Proof of Stake Shard
 
 The proof of stake (PoS) shard, provides users the ability to have different amount of voting power/blocks produced in the network. PoS Network this can be customised based on the preferences of the network owners.Validators are required to lock up a certain amount of cryptocurrency or token as collateral, which incentivizes them to act honestly and secure the network. PoS shards are designed to be more energy-efficient, secure, and decentralized than PoW shards, as they do not require large amounts of computational power and electricity to perform the validation process.
```
[the remark this answers is not in the document]
Comment (chefsale) [reply docs/introduction/architecture.md:30]: Make this more clear is a bit of a tidious feedback. Lets try to add more concrete examples of what needs to be more clear and how.

File: docs/introduction/architecture.md [hunk 3]
```diff
@@ -3,51 +3,51 @@ title: Shard Architecture
...
-For example, if a private shard is run by 100 validators owned by 100 different companies, it can be configured so that every single one of them has 1 percent of the total stake distributed. In this case every company will produce 1% of the blocks and more than 66% of the companies will need to behave maliciously and collude to corrupt the shard and 33% + 1 to stall the network.
-In the future we plan to implement different validator staking mechanisms. Being an owner of an NFT could be an entry ticket to become an validator in the shard. The NFT can represent anything from ownership of a game to having gone through KYC.
+Private shard validators hold a portion of the tokens allocated at the beginning of the shard. These tokens are solely used for achieving consensus and do not possess any real monetary value. By default, they are designed for networks without actual monetary transactions. However, it is possible to create networks with monetary value, such as in the case of a Central Bank Digital Currency (CBDC).
```
Comment (emojapaula) [inline docs/introduction/architecture.md:18]: allocated at the shard creation.

Comment (emojapaula) [reply docs/introduction/architecture.md:18]: At least one validator is needed for the private shard to work properly, but it is recommended to have at least four validators for redundancy and decentralisation

File: docs/introduction/architecture.md [hunk 4]
```diff
@@ -3,51 +3,51 @@ title: Shard Architecture
...
+In a proof of authority (PoA) shard, all validators have equal voting power and an equal number of blocks. PoA shards are commonly used in private or permissioned blockchain networks where validators are known and trusted entities, such as consortium members or enterprise partners. PoA shards offer faster transaction speeds and lower energy consumption compared to proof of work (PoW) or proof of stake (PoS) consensus mechanisms.
 
 ## Proof of Stake Shard
 
 The proof of stake (PoS) shard, provides users the ability to have different amount of voting power/blocks produced in the network. PoS Network this can be customised based on the preferences of the network owners.Validators are required to lock up a certain amount of cryptocurrency or token as collateral, which incentivizes them to act honestly and secure the network. PoS shards are designed to be more energy-efficient, secure, and decentralized than PoW shards, as they do not require large amounts of computational power and electricity to perform the validation process.
```
Comment (emojapaula) [inline docs/introduction/architecture.md:35]: Should network in PoS Network be capitalized

Comment (emojapaula) [reply docs/introduction/architecture.md:35]: There is a space missing before Validators

Comment (emojapaula) [reply docs/introduction/architecture.md:35]: as a collateral

Comment (emojapaula) [reply docs/introduction/architecture.md:35]: to act honestly.

File: docs/introduction/architecture.md [hunk 5]
```diff
@@ -3,51 +3,51 @@ title: Shard Architecture
...
+In a proof of authority (PoA) shard, all validators have equal voting power and an equal number of blocks. PoA shards are commonly used in private or permissioned blockchain networks where validators are known and trusted entities, such as consortium members or enterprise partners. PoA shards offer faster transaction speeds and lower energy consumption compared to proof of work (PoW) or proof of stake (PoS) consensus mechanisms.
 
 ## Proof of Stake Shard
 
 The proof of stake (PoS) shard, provides users the ability to have different amount of voting power/blocks produced in the network. PoS Network this can be customised based on the preferences of the network owners.Validators are required to lock up a certain amount of cryptocurrency or token as collateral, which incentivizes them to act honestly and secure the network. PoS shards are designed to be more energy-efficient, secure, and decentralized than PoW shards, as they do not require large amounts of computational power and electricity to perform the validation process. 
 
-## Migration from the shard from PoA to PoS
+## Migration from the shard from Proof of Authority (PoA) to Proof of Stake (PoS)
```
Comment (emojapaula) [inline docs/introduction/architecture.md:37]: I dont understand this title, is it maybe migration of the shard

File: docs/introduction/architecture.md [hunk 6]
```diff
@@ -3,51 +3,51 @@ title: Shard Architecture
...
 
 ## State and Execution
 
-The private execution and state is accessible only by the nodes inside the private shard. Only information which is publicly announced are the light client proof for private shard blocks, used by the light client, prover and connector contracts.
-Additionally, some contract state may be accessible from a public chain if the developer allowed such access in the contract, through the bridge with cross shard contract calls which is one of main features of Calimero shards.
+In a private shard, only the nodes within that shard can access and see the execution and state. The only information made public is the light client proof for private shard blocks. This proof is used by certain contracts to verify block validity. Additionally, if the contract developer allows it, some contract states may be accessible from a public chain through cross-shard contract calls using the bridge feature in Calimero shards.
```
Comment (emojapaula) [inline docs/introduction/architecture.md:43]: execution and the state

Comment (emojapaula) [reply docs/introduction/architecture.md:43]: in Calimero console. i think

**Review by emojapaula — CHANGES_REQUESTED** [review]

**Comment by chefsale** [comment]
@emojapaula @fakela Can we land this?

File: docs/introduction/architecture.md (hunk 4 above)
Comment (fakela) [reply docs/introduction/architecture.md:35]: > as a collateral

"as collateral" is more commonly used and generally preferred.

Comment (fakela) [reply docs/introduction/architecture.md:35]: > to act honestly.



> to act honestly.

I think the full sentence is okay as well,  no need to add the stop`.`  at honestly

**Review by emojapaula — APPROVED** [review]

**Pull request merged**